config/application.rb

```diff
@@ -136,6 +136,13 @@ class Application < Rails::Application
     # Enable escaping HTML in JSON.
     config.active_support.escape_html_entities_in_json = true
 
+    # Don't raise exception for common parameters
+    config.action_controller.always_permitted_parameters = %w(
+      controller action format locale utf8 _method authenticity_token commit redirect
+      page per_page paginate search order sort sort_by sort_order
+      _ie_support fakepassword apiv id organization_id location_id user_id
+    )
```

Comment:
From the security perspective, the following can be theoretically dangerous: id organization_id location_id user_id but I think it is worth taking the risk - will unlikely happen and your patch is a great security improvement.

Reply:
This doesn't actually permit these params, despite the naming. it just prevents raising an exception when these params are found and permit! is called on them.